src/set_theory/zfc.lean

```diff
@@ -313,7 +323,10 @@ end resp
   function. This isn't exactly definability, but is useful as a sufficient
   condition for functions that have a computable image. -/
 class inductive definable (n) : arity Set.{u} n → Type (u+1)
-| mk (f) : definable (resp.eval _ f)
+| mk (f) : definable (resp.eval n f)
+
+-- Todo (Vi): this shouldn't be a class, since sets have lots of nontrivial equalities between them.
```

Comment:
See [Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/ZFC.20definable.20class/near/289072024) for discussion of this remark.

Reply:
The consensus here seems to be that we should transition away from typeclasses, but eventually have enough theorems (and perhaps some metaprogramming) so that the definability of any formula can be near-automatically deduced.

Reply:
I'd rather not have to read a zulip thread to decide whether I agree with TODO comment; can't you add the TODO comments in a separate PR to the minor cleanups?

Reply:
Sure, I'll remove them.